packages/runtime-dom/src/helpers/useCssVars.ts

```diff
@@ -26,7 +26,13 @@ export function useCssVars(getter: (ctx: any) => Record<string, string>) {
     return
   }
 
-  const updateTeleports = (instance.ut = (vars = getter(instance.proxy)) => {
+  const updateTeleports = (instance.ut = (
+    vars = getter(instance.proxy),
+    hydration = false,
```

Comment:
I don't think this is a good idea to use updateTeleports to get cssVars. It is only used to apply cssVars inside Teleport.
Maybe you can do something like this
```js
instance.getCssVars=()=>getter(instance.proxy)
```